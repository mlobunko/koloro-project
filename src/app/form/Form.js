import React from "react";

import { StyledForm, StyledButton, StyledInput } from "./styles";

export const Form = ({ landscape }) => (
  <StyledForm landscape={landscape}>
    <StyledInput type={"text"} placeHolder={"Имя"} />
    <StyledInput type={"tel"} placeHolder={"Телефон"} />
    <StyledInput type={"email"} placeHolder={"E-mail"} />
    <StyledButton ariaLabel={"submit form button"} type={"submit"}>
      КОЛОРОНИРОВАТЬСЯ
    </StyledButton>
  </StyledForm>
);

export default Form;
