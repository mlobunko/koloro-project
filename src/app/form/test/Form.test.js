import React from "react";
import { shallow } from "enzyme";

import { Form } from "../Form";

describe("render", () => {
  it("renders correctly", () => {
    const landscape = true;
    const wrapper = shallow(<Form landscape={landscape} />);
    expect(wrapper).toMatchSnapshot();
  });
});
