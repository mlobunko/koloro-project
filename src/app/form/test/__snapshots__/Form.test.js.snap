// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render renders correctly 1`] = `
<StyledForm
  landscape={true}
>
  <StyledInput
    placeHolder="Имя"
    type="text"
  />
  <StyledInput
    placeHolder="Телефон"
    type="tel"
  />
  <StyledInput
    placeHolder="E-mail"
    type="email"
  />
  <styled.button
    ariaLabel="submit form button"
    type="submit"
  >
    КОЛОРОНИРОВАТЬСЯ
  </styled.button>
</StyledForm>
`;
